src/Analyzers/Core/Analyzers/EnforceOnBuildValues.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CodeStyle;
+
+namespace Microsoft.CodeAnalysis.Diagnostics
+{
+    internal static class EnforceOnBuildValues
```

Comment:
NOTE: This is the core file that needs to be reviewed for the default code style analyzer bucketing. The values selected here match the P1/P2/P3 buckets identified for code style build enforcement in the internal email thread with the IDE team. These grouping are pretty likely to change in future as we get more customer feedback and iterate towards ideal groups.